molo/profiles/templates/profiles/viewprofile.html

```diff
@@ -3,22 +3,25 @@
 {% get_settings %}
 
 {% block content %}
+<h1 class="heading heading--profiles">{% trans "Hello " %}{{ user.username }}</h1>
+<div class="profiles">
+  <p>{% trans "Display Name: " %}{{ user.profile.alias | default:"Anonymous" }}</p>
+  <p>{% trans "Date of Birth: " %}{{ user.profile.date_of_birth | date:"M d, Y" }}</p>
+    {% if settings.profiles.UserProfilesSettings.show_mobile_number_field %}
+      <label for="id_profile.mobile_number">{% trans "Mobile Number: " %}{{ user.profile.mobile_number | default:"" }}</label>
+    {% endif %}
+    {% if settings.profiles.UserProfilesSettings.show_email_field %}
+      <label for="id_email">{% trans "Email Address: " %}{{ user.email | default:"" }}</label>
+    {% endif %}
 
-<div class="profile-style">
-
-  <h3>{% trans "Hello " %}{{ user.username }}</h3>
-  <label class="label-wide">{% trans "Display Name: " %}{{ user.profile.alias | default:'Anonymous' }}</label>
-  <label class="label-wide">{% trans "Date of Birth: " %}{{ user.profile.date_of_birth | date:"M d, Y" }}</label>
-  {% if settings.profiles.UserProfilesSettings.show_mobile_number_field %}
-    <label class="label-wide">{% trans "Mobile Number: " %}{{ user.profile.mobile_number | default:'' }}</label>
-  {% endif %}
-  {% if settings.profiles.UserProfilesSettings.show_email_field %}
-    <label class="label-wide">{% trans "Email Address: " %}{{ user.email | default:'' }}</label>
-  {% endif %}
-  <a href="{% url 'molo.profiles:auth_logout' %}" class="btn btn-wide"><input type="button" value='{% trans "log out" %}' class="btn btn-wide" /></a>
-  <a href="{% url 'molo.profiles:edit_my_profile' %}" class="btn btn-wide"><input type="button" value='{% trans "Edit My Profile" %}' class="btn btn-wide" /></a>
-  <a href="{% url 'molo.profiles:profile_password_change' %}" class="btn btn-wide"><input type="button" value='{% trans "Change My Password" %}' class="btn btn-wide" /></a>
-
+    <a href="{% url "molo.profiles:auth_logout" %}">
```

Comment:
Hmmm... Can we call Log out, Edit Profile, and Change Password `class="profile__nav-item"`